canvas_modules/common-canvas/src/command-actions/commonPropertiesAction.js

```diff
@@ -37,9 +41,23 @@ export default class CommonPropertiesAction extends Action {
 		this.applyPropertyChanges(this.newValues.properties, this.appData, this.newValues.additionalInfo, this.newValues.undoInfo, this.newValues.uiProperties);
 	}
 
-	// TODO - Implement this to get the command action label from translation
-	// file or call a getActionLabel callback.
 	getLabel() {
-		return "";
+		if (this.propertiesActionLabelHandler) {
+			const label = this.propertiesActionLabelHandler();
+			if (label) {
+				return label;
+			}
+		}
+		return this.getDefaultLabel("action.commonProperties");
```

Comment:
Since the translated text is part of `common-canvas/locales/command-actions/locales/en.json`, you dont need to pass in the `intl` object from common-properties. You can use the same `labelUtil.getLabel()` to get the default label

Reply:
@matthoward366 Do you have any input on this? 
Its not clear where the separation is between canvas and properties with this action. It doesnt make sense to import `labelUtil` in properties and pass that into the action just to get the default label, which is added in the canvas translation file. So maybe my previous comment doesnt make sense..

Reply:
We don't want to use `labelUtil` since that is canvas specific.  I would think we'd want to pass in the label into the action.  Then we wouldn't pass `propertiesActionLabelHandler, intl` in.  We'd call the callback when the action is added to the stack.  If we do it here the consumer app wouldn't really know which label to get.